api/src/client/ApiFormationHealth.ts

```diff
@@ -46,6 +46,18 @@ export const ApiFormationHealth: UserData = {
           { identifier: "some-identifier-15484222", dueDate: "2024-02-09", calendarEventType: "TAX-FILING" },
         ],
       },
+      taxClearanceCertificateData: {
```

Comment:
A high value add here might be to use `createEmptyBusiness` and just override the values that we need for the health check so that we don't have to update the shape of `userData` every time.

Not setting a value because I don't think you need to do this as part of the work.

Reply:
I may knock this out in a separate PR so that we could remove changes to this file.

Reply:
merged

Reply:
thanks jp

Reply:
added pivitol ticket to the PR